Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -287,6 +287,9 @@ void UpdateTextColor()
 
 		void UpdatePadding()
 		{
+			if (!Element.IsSet(Button.PaddingProperty))
+				return;
+
 			Control?.SetPadding(
 				(int)(Context.ToPixels(Element.Padding.Left) + _paddingDeltaPix.Left),
 				(int)(Context.ToPixels(Element.Padding.Top) + _paddingDeltaPix.Top),
```

Comment:
I don't think this covers the case when a `Padding` is set, and then cleared.

Reply:
Is it possible to clear padding? It can not be null, and if set to `new Thickness()`, it will be set to `0`.
but if cleaning is possible then need `SetPadding` to default value 
```C# 
Control?.SetPadding(_defaultPadding.Left, _defaultPadding.Top, _defaultPadding.Right, _defaultPadding.Bottom);
```

Reply:
@paymicro Yep, you can do `button.ClearValue(Button.PaddingProperty)`. You and Stephane are describing the pattern that we generally follow for such property (i.e., store the default value and restore when the value is unset), and I *did* think about it yesterday, but I thought to myself...who would do such a thing? Me being lazy. :D fixing now